archetypes/se/basic/src/main/resources/pom.xml.mustache

```diff
@@ -7,36 +6,26 @@
     <parent>
         <groupId>io.helidon.applications</groupId>
         <artifactId>helidon-se</artifactId>
-        <version>@project.version@</version>
-        <relativePath/>
+        <version>{{helidonVersion}}</version>
     </parent>
-    <groupId>${groupId}</groupId>
-    <artifactId>${artifactId}</artifactId>
-    <version>${version}</version>
-    <name>${artifactId}</name>
+    <groupId>{{groupId}}</groupId>
+    <artifactId>{{artifactId}}</artifactId>
+    <version>{{version}}</version>
+    <name>{{name}}</name>
 
     <properties>
-        <mainClass>${package}.Main</mainClass>
+        <mainClass>{{package}}.Main</mainClass>
     </properties>
 
     <dependencies>
         <dependency>
-            <groupId>io.helidon.webserver</groupId>
-            <artifactId>helidon-webserver</artifactId>
+            <groupId>io.helidon.bundles</groupId>
```

Comment:
Please stop using this bundle.
I think it is confusing - it just adds support for JSON-P.